httpcache_test.go

```diff
@@ -23,22 +26,25 @@ func TestMaxAge(t *testing.T) {
 	}`, "caddyfile")
 
 	resp1, _ := tester.AssertGetResponse(`http://localhost:9080/cache-max-age`, 200, "Hello, max-age!")
-	if resp1.Header.Get("Cache-Status") != "Caddy; fwd=uri-miss; stored" {
-		t.Errorf("unexpected Cache-Status header %v", resp1.Header.Get("Cache-Status"))
+	if resp1.Header.Get("Cache-Status") != "Souin; fwd=uri-miss; stored" {
+		t.Errorf("unexpected Cache-Status header %v", resp1.Header)
 	}
 
 	resp2, _ := tester.AssertGetResponse(`http://localhost:9080/cache-max-age`, 200, "Hello, max-age!")
-	if resp2.Header.Get("Cache-Status") != "Caddy; hit" {
+	if resp2.Header.Get("Cache-Status") != "Souin; hit; ttl=1" {
 		t.Errorf("unexpected Cache-Status header %v", resp2.Header.Get("Cache-Status"))
 	}
 }
 
 func TestSMaxAge(t *testing.T) {
 	tester := caddytest.NewTester(t)
-	tester.InitServer(` 
+	tester.InitServer(`
 	{
 	  http_port     9080
 	  https_port    9443
+	  cache {
+		  ttl 1000s
```

Comment:
This should be optional. We should use a heuristic (https://httpwg.org/specs/rfc7234.html#rfc.section.4.2.2) if a specific TTL isn't set. Varnish uses 120s by default.

Reply:
This should be related to the core then.

Reply:
As it been fixed in core?

Reply:
Yes it is :)